CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 1.8.0 / 2017-10-06
+
+* [CHANGE] Rule links link to the _Console_ tab rather than the _Graph_ tab to
+  not trigger expensive range queries by default.
+* [CHANGE] Changed help link to `https://prometheus.io/docs`.
+* [FEATURE] Ability to act as a remote read server for other Prometheus servers.
+* [FEATURE] K8s SD: Support discovery of ingresses.
+* [FEATURE] Consul SD: Support for node metadata.
+* [FEATURE] Openstack SD: Support discovery of hypervisors.
+* [FEATURE] Expose current Prometheus config via `/status/config`.
+* [FEATURE] Allow to collapse jobs on `/targets` page.
+* [FEATURE] Add `/-/healthy` and `/-/ready` endpoints.
+* [FEATURE] Add color scheme support to console templates.
+* [ENHANCEMENT] Remote storage connections use HTTP keep-alive.
+* [ENHANCEMENT] Improved logging about remote storage.
+* [ENHANCEMENT] Failure to send NaN values to remote storage logged on debug
```

Comment:
This is in the example remote adapter, I'm not sure we should list it in the main changelog

Reply:
Removed.